src/App.tsx

```diff
@@ -41,7 +41,6 @@ function App() {
         { path : "requsetRestPass", element: <RequsetResetPass/> },
```

Comment:
It is recommended  for URL names to be separated by hyphens rather than capitalizing each word. For example if you clicked on `requestResetPass`, you 'll notice that your device sees it as only one word, but when changed to `request-reset-password`, every word can be selected separately. You 'll see this convention in mostly any professional website. Here is Next.js for example:
![image](https://github.com/AhmedEl-Hussin/Quize-App/assets/64327685/0772f705-64a6-4aeb-b890-60ed95ca8e1f)